lib/wallet/http.js

```diff
@@ -1160,12 +1160,13 @@ class HTTP extends Server {
 
       assert(broadcast ? sign : true, 'Must sign when broadcasting.');
 
+      const options = TransactionOptions.fromValidator(valid);
+
       if (!name) {
-        const tx = await req.wallet.sendRedeemAll();
+        const tx = await req.wallet.sendRedeemAll({passphrase, ...options});
```

Comment:
I think this does not respect `broadcast` option.

We could have `const mtx = await req.wallet.createRedeemAll(options);` and do the same thing as we do for mtx bellow.

Can be done bellow where `mtx` is defined, make it `let mtx` and create that MTX based on `!name`.

Reply:
Great catch, thanks -- will take this suggestion and push